Dockerfile

```diff
@@ -1,13 +1,13 @@
-## Installs OpenJDK17 and openssl (used by Swirlds Platform to 
-## generate node keys for e.g. signing states), then copies 
+## Installs OpenJDK17 and openssl (used by Swirlds Platform to
+## generate node keys for e.g. signing states), then copies
 ## required libraries and startup assets for a node with:
-##  * Configuration from /opt/hedera/services/config-mount; and, 
-##  * Logs at /opt/hedera/services/output; and, 
+##  * Configuration from /opt/hedera/services/config-mount; and,
+##  * Logs at /opt/hedera/services/output; and,
 ##  * Saved states under /opt/hedera/services/output
 FROM ubuntu:21.10 AS base-runtime
 RUN apt-get update && \
     apt-get upgrade -y && \
-    apt-get install -y dos2unix openssl libsodium23 postgresql-client
+    apt-get install -y dos2unix openssl libsodium23 bind9-host
```

Comment:
Do we still need libsodium23?

Reply:
Potentially depending on what Oleg finds in the Platform SDK v0.28.0 memory pressure issue he was investigating. Early analysis showed that bouncy castle was creating a bunch of `X25519Field` instances in the `Signature.verify()` method. JVM implementation of X25519/ED25519 was insanely slow from a computational standpoint. Our only recourse may be to go back to LibSodium/LazySodium.